packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -170,29 +160,67 @@ const DomainPicker: FunctionComponent< Props > = ( {
 							<DomainCategories selected={ domainCategory } onSelect={ setDomainCategory } />
 						</div>
 					) }
-					<div className="domain-picker__suggestion-item-group">
-						{ domainSuggestions?.map( ( suggestion, i ) => (
-							<SuggestionItem
-								key={ suggestion.domain_name }
-								suggestion={ suggestion }
-								railcarId={ baseRailcarId ? `${ baseRailcarId }${ i }` : undefined }
-								isRecommended={ i === 0 }
-								onRender={ () =>
-									handleItemRender( suggestion, `${ baseRailcarId }${ i }`, i, i === 0 )
-								}
-								onSelect={ onDomainSelect }
-							/>
-						) ) ?? times( quantity, ( i ) => <SuggestionItemPlaceholder key={ i } /> ) }
-						{ ! isExpanded &&
-							allDomainSuggestions?.length &&
-							allDomainSuggestions?.length > quantity && (
-								<div className="domain-picker__show-more">
-									<Button onClick={ () => setIsExpanded( true ) }>
-										{ __( 'View more results' ) }
-									</Button>
+					{ domainSuggestions ? (
```

Comment:
This ternary wouldn't be necessary if we render the placeholders exactly where the items are rendered when `domainSuggestions` are loaded.